packages/ui/src/components/CardRepeater/Card.tsx

```diff
@@ -169,19 +147,15 @@ const Card = ({ index, edit, remove, error, onMove, children }: CardProps) => {
           )}
           {/* INDEX */}
           {!hideIndex && (
-            <span
-              aria-hidden="true"
-              data-h2-text-align="base(center)"
-              data-h2-font-weight="base(700)"
-            >
+            <span aria-hidden="true" className="text-center font-bold">
               {position}
             </span>
           )}
           {/* DOWN ARROW */}
           {!disableIncrement ? (
             <Action
               onClick={increment}
-              animation={!shouldReduceMotion ? "translate-up" : "none"}
+              animation={!shouldReduceMotion ? "translate-down" : "none"}
```

Comment:
Is this supposed to be opposite to before?

Reply:
The animation looks ok to me.  :shrug:

Reply:
Oh, I see, it was wrong before.  :laughing:

Reply:
Yeah, at first I was like "why do we even have the down if all we ever us is up?" then I realized it was probably a copy-paste mistake :stuck_out_tongue: